src/mocks/cardData.ts

```diff
@@ -0,0 +1,51 @@
+interface Card {
+    "default-image-for-card": string;
+    "default-film-name": string;
+    "default-film-date": string;
+    "film-rating": number;
```

Comment:
in what format does the backend return the data?

Reply:
Should be interface with the fields that UI expects from the backend

Reply:
used field names like in JSON which returns backend